tests/test_objective_funs.py

```diff
@@ -1598,6 +1600,41 @@ def test_compute_scalar_resolution():  # noqa: C901
         f[i] = obj.compute_scalar(obj.x(eq))
     np.testing.assert_allclose(f, f[-1], rtol=1e-2)
 
+    # TODO: add test for PlasmaVesselDistance
+
+    # Omnigenity
+    # (needs an approximately QP equilibrium and field)
+    surf = FourierRZToroidalSurface.from_qp_model(
+        major_radius=1,
+        aspect_ratio=20,
+        elongation=6,
+        mirror_ratio=0.2,
+        torsion=0.1,
+        NFP=1,
+        sym=True,
+    )
+    eq = Equilibrium(Psi=6e-3, M=4, N=4, surface=surf)
+    eq, _ = eq.solve(objective="vacuum", verbose=3)
+    field = OmnigenousField(
+        L_well=0,
+        M_well=2,
+        L_shift=0,
+        M_shift=0,
+        N_shift=0,
+        NFP=eq.NFP,
+        helicity=(0, eq.NFP),
+        B_lm=np.array([0.8, 1.2]),
+    )
+    f = np.zeros_like(res_array, dtype=float)
+    for i, res in enumerate(res_array + 0.5):  # omnigenity needs higher resolution
+        grid = LinearGrid(M=int(eq.M * res), N=int(eq.N * res), NFP=eq.NFP)
```

Comment:
`int` does not round, it just discards the part past the `int`, use `round` instead

Reply:
Rounding up here isn't necessary (the resolution is already sufficient), the point is only to make sure it gets cast to an integer.